extension/sqlite/src/sqlite_connector.cpp

```diff
@@ -0,0 +1,18 @@
+#include "sqlite_connector.h"
+
+namespace kuzu {
+namespace sqlite_extension {
+
+void SqliteConnector::connect(const std::string& dbPath, const std::string& catalogName,
+    main::ClientContext* /*context*/) {
+    // Creates an in-memory duckdb instance, then install httpfs and attach remote SQLITE.
+    instance = std::make_unique<duckdb::DuckDB>(nullptr);
+    connection = std::make_unique<duckdb::Connection>(*instance);
+    executeQuery("install sqlite;");
+    executeQuery("load postgres;");
```

Comment:
Why `load postgres`? hmm why isn't there any tests failing due to this ? 😅

Reply:
I think duckdb doesn't have to load sqlite in order to read sqlite database.